backend/ai_firm/ceo.py

```
@@ -0,0 +1,312 @@
+        personality_reasoning = {
+            CEOPersonality.CONSERVATIVE: "Prioritizing capital preservation",
+            CEOPersonality.AGGRESSIVE: "Focusing on growth opportunities",
+            CEOPersonality.BALANCED: "Balancing risk and opportunity",
+            CEOPersonality.ADAPTIVE: "Adapting strategy to current conditions"
+        }
+        
+        reasoning_parts.append(personality_reasoning[self.personality])
+        
+        return ". ".join(reasoning_parts) + "."
+    
+    def _assess_impact(self, context: Dict, confidence: float) -> str:
```

Ruff (0.12.2)

Unused method argument: `context`

(ARG002)